copy_cat/settings/common.py

```diff
@@ -18,7 +18,7 @@
     # External
     'storages',
     # Local
-    'forum'
+    'forum.apps.ForumConfig'
```

Comment:
Why did you change this? You shouldn't need to.

Reply:
its something that was in the tutorial that i thought may be related to overall app.  Changed back and it still works!